src/main/scala/mirroring/services/databases/JdbcCTService.scala

```diff
@@ -48,7 +48,7 @@ class JdbcCTService(jdbcContext: JdbcContext) extends Serializable {
       val schema: StructType = buildStructFromResultSet(resultSet)
       log.debug(schema)
       log.info("Executing procedure to create rdd...")
-      val myRDD: JavaRDD[Array[Object]] = JdbcRDD.create(
+      var myRDD: JavaRDD[Array[Object]] = JdbcRDD.create(
```

Comment:
I think it may be refactored as
```scala
      val myRDD: JavaRDD[Array[Object]] = JdbcRDD
        .create(
          getSparkSession.sparkContext,
          connectionManager,
          jdbcContext.ctChangesQuery,
          params(0).toLong,
          params(1).toLong,
          1,
          r => JdbcRDD.resultSetToObjectArray(r)
        ).repartition(getSparkSession.conf.get("spark.sql.shuffle.partitions").toInt)
```
Let's refactor it in the next iteration,